### PR TITLE
Fixed links in the README after renaming folders.

### PR DESCRIPTION
**Pull request by aputriax**
Signed-off-by: Anton Putria <antonx.putria@intel.com>

### CODE REVIEW

File: test/test-cases/scale/vnet2vnet/README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ The files found within this directory serve the following purpose:
 
 # test cases
 
-| Test case                                      | Description                                               |
-| ---------------------------------------------- | --------------------------------------------------------- |
-| [vxlan_1vpc_1ip](README.vxlan_1vpc_1ip.md)     | performance numbers for best case scenario                |
+| Test case                                      | Description                                                    |
+| ---------------------------------------------- | -------------------------------------------------------------- |
+| [vxlan_1vpc_1ip](one-ip/README.md)             | performance numbers for best case scenario                     |
+| [vxlan_8vpc_48K-ips](48K-ips/README.md)        | performance with an objective of maintaining 6M parallel flows |
```
Comment (chrispsommers) [inline test/test-cases/scale/vnet2vnet/README.md:101]: This link appears to be broken although it "looks" correct.

**Review by chrispsommers — CHANGES_REQUESTED** [review]
I think there's one broken link, I tried clicking it

File: test/test-cases/scale/vnet2vnet/README.md (hunk 1 above)
Comment (aputriax) [reply test/test-cases/scale/vnet2vnet/README.md:101]: This one looks good, I can follow it. But the same one on the line 15 I will fix it.

**Comment by lguohan** [comment]
looks like the broken link is fixed. @chrispsommers , can you check and approve?

**Review by chrispsommers — APPROVED** [review]
LGTM, thanks!

**Comment by chrispsommers** [comment]
Links are fixed and new links were added per request, thanks!

**Pull request merged**